storyblok/storyblok-design-system
# feat(work-1475): add vertical-menu icon

Add **vertical-menu** icon

## Pull request type

Jira Link: [WORK-1475](https://storyblok.atlassian.net/browse/WORK-1475)




## How to test this PR
Go to SbIcon and check if the icon exists.
![Captura de Tela 2023-12-13 às 11 33 42](https://github.com/storyblok/storyblok-design-system/assets/8209305/116d05b2-cb72-4143-b238-56ee7060dfe0)




[WORK-1475]: https://storyblok.atlassian.net/browse/WORK-1475?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
